src/components/FileList.tsx

```diff
@@ -914,19 +1156,21 @@ export class FileList extends React.Component<IFileListProps, IFileListState> {
    * @param file File to open diff for
    * @param currentRef the ref to diff against the git 'HEAD' ref
    */
-  private async _openDiffView(file: Git.IStatusFile): Promise<void> {
-    try {
-      await this.props.commands.execute(ContextCommandIDs.gitFileDiff, {
-        files: [
-          {
-            filePath: file.to,
-            isText: !file.is_binary,
-            status: file.status
-          }
-        ]
-      } as CommandArguments.IGitFileDiff as any);
-    } catch (reason) {
-      console.error(`Failed to open diff view for ${file.to}.\n${reason}`);
-    }
+  private _openDiffViews(files: Git.IStatusFile[]): void {
+    files.forEach(file => {
+      try {
+        this.props.commands.execute(ContextCommandIDs.gitFileDiff, {
+          files: [
+            {
+              filePath: file.to,
+              isText: !file.is_binary,
+              status: file.status
+            }
+          ]
+        } as CommandArguments.IGitFileDiff as any);
+      } catch (reason) {
+        console.error(`Failed to open diff view for ${file.to}.\n${reason}`);
+      }
```

Comment:
As the `execute` is async, you should either await it or define `.catch` to that call:

```suggestion
        this.props.commands.execute(ContextCommandIDs.gitFileDiff, {
          files: [
            {
              filePath: file.to,
              isText: !file.is_binary,
              status: file.status
            }
          ]
        } as CommandArguments.IGitFileDiff as any)
       .catch((reason) => {
        console.error(`Failed to open diff view for ${file.to}.\n${reason}`);
      });
```